woocommerce/google-listings-and-ads
# Update Google Ads credit incentive info in plugin UI

### Changes proposed in this Pull Request:

Closes #1214 .

This PR updates the Google Ads ad credit incentives in various parts of the plugin from Spend Match $150 to Spend Get $500. The fulfillment period is changed from 30 days to 60 days.

Internal link: pcTzPl-eI-p2 - Search for "Messaging Update Tracker" and "coupon amount per country".

### Screenshots:

FAQ in Get Started page:

![image](https://user-images.githubusercontent.com/417342/151396008-df627e7c-65e4-45ad-9003-1c19a0b17998.png)

Submission success guide, after setting up free listings:

![image](https://user-images.githubusercontent.com/417342/150991093-dd6c82ba-8795-44ab-a968-052f791118b5.png)

Performance (Paid Campaigns) card in dashboard:

![image](https://user-images.githubusercontent.com/417342/150991176-386e97cf-6291-4308-a184-fb60a4de60a0.png)

Free credit eligibility box in Setup Ads flow, showing Spend Get $500 and the 60 days period. This will only be displayed for sub-account that is newly created within the last 60 days: 

![image](https://user-images.githubusercontent.com/417342/151302091-1302f5ce-e2c3-4fe5-9cd4-641fbd5e8e03.png)

Free credit modal in Setup Ads flow:

![image](https://user-images.githubusercontent.com/417342/150991234-be6b46bd-f1d8-44c8-b8d0-68b893865a3b.png)

Note: For screenshots prior to the changes, refer to issue #1214. 

### Detailed test instructions:

Note: Please visit the internal link pcTzPl-eI-p2 and have the documents "Messaging Update Tracker" and "coupon amount per country" opened for your testing reference.

1. Install and activate the plugin and go to the plugin page. You should see a page with FAQs at the bottom. The FAQs should contain ad credit offer info like the above screenshot.
2. Complete the Setup MC flow, and you should see the submission success guide. The UI should match the expected result in Messaging Update Tracker document.
2. Go to the Dashboard page. You should see the "Performance (Paid Campaigns)" card. It should should match the expected result in Messaging Update Tracker document.
3. Proceed to add a new paid campaign, and go through the Setup Ads flow until Step 2.
4. In Setup Ads Step 2, in the Budget section, you should see a green-colored "You’re eligible for free ad credit!" box. The box should be as per above screenshot.
    - If you don't see it (e.g. you might have connected an existing old Google Ads account), you can modify the following line to `{ ( hasFreeAdCredit || true ) && <FreeAdCredit /> }` to force the box to show up.

https://github.com/woocommerce/google-listings-and-ads/blob/6a2bb66ef3787927ef61829b9e7bc4cf282356db/js/src/components/paid-ads/budget-section/index.js#L72

5. The green box should mention "60 days" period and not "30 days".
6. Click on the "here" link in the green box. The "Check your maximum free credit" modal should show up. Verify that the country, amount and currency are correct as per the "coupon amount per country" document.



### Changelog entry

> Update - Update Google Ads credit incentive info in plugin UI.
